ClientCore/ClientConfiguration.cs

```diff
@@ -305,10 +305,11 @@ public string GetGameExecutableName()
 
         public OSVersion GetOperatingSystemVersion()
         {
-            Version osVersion = Environment.OSVersion.Version;
-
             if (Environment.OSVersion.Platform == PlatformID.Win32NT)
             {
+#if NETFRAMEWORK
```

Comment:
If no manifest files are present, the OSVersion will stay at 6.2 even if the actual OS is Win8/10/11. By specifying the OS id in the manifest file the behavior will change.

Reply:
Updated the OS detection code as it broke with the added OS versions in the manifest.